### ISSUE TITLE
Slam sucks balls

### ISSUE THREAD

**Issue by radioradioradio** [cybernine186/quests#16]
Shamans slam misses 29/30 times in testing. Likely related to not having any skill in bash. Need to pump those numbers up.

Knuckle — Yesterday at 2:23 PM
think slam on shaman was roughly 15-20% hit rate back on vztz 1.0 days
20% sounds about right maybe even 25%
was never something you could count on but could usually land one in a fight

Code paths are kinda janky and bash/slam get boiled down into just bash basically, probably hardcode a reasonable hit chance for if Race is Ogre/Troll/Barb.

Basically copy this code here into the end of Mob::GetTotalToHit(), bump accuracy up to whatever if the conditions meet, std::max() the skill check vs base value.
https://github.com/EQEmu/Server/blob/10083387b6009c27678c10c2c7e2bd6d081af0d1/zone/special_attacks.cpp#L235
